nebula_decoders/src/nebula_decoders_continental/decoders/continental_ars548_decoder.cpp

```diff
@@ -245,6 +245,16 @@ bool ContinentalARS548Decoder::parse_detections_list_packet(
 bool ContinentalARS548Decoder::parse_objects_list_packet(
   const nebula_msgs::msg::NebulaPacket & packet_msg)
 {
+  // cSpell:ignore knzo25
+  // NOTE(knzo25): In the radar firmware used when developing this driver,
+  // corner radars were not supported. When a new firmware addresses this,
+  // the driver will be updated.
+  if (
+    std::abs(radar_status_.yaw) > 5.0 * M_PI / 180.0 &&
+    std::abs(radar_status_.yaw) < 90.0 * M_PI / 180.0) {
```

Comment:
For angle conversions, could you use the `deg2rad()` function for clarity?
Also, since this check occurs in three places, could you factor it out into a `is_corner_radar(yaw)` function?

Reply:
Thanks ! 
Addressed in [99ccb0f](https://github.com/tier4/nebula/pull/252/commits/99ccb0f84ff7af32a4a7eef7edf84e3f533a68a4)